records_mover/records/targets/table/move_from_dataframes_source.py

```diff
@@ -30,25 +30,29 @@ def __init__(self,
         super().__init__(prep, target_table_details, processing_instructions)
 
     def move(self) -> MoveResult:
-        if len(self.table_target.known_supported_records_formats()) != 0:
-            if self.table_target.can_move_from_fileobjs_source():
-                return self.move_from_dataframes_source_via_fileobjs()
-            else:
-                # Some databases, like Redshift, can't load from a
-                # stream, but can load from files on an object store
-                # they're pointed to.
-                return self.move_from_dataframes_source_via_records_directory()
+        # TODO: Clean up redundant clause
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker